source/retryable-reads/tests/README.rst

```diff
@@ -156,6 +156,9 @@ the SDAM's ``minHeartbeatFrequencyMS`` from 500ms to 50ms, thus decreasing the
 waiting time after a "not master" error:
 
 .. _minHeartbeatFrequencyMS: https://github.com/mongodb/specifications/blob/master/source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst#minheartbeatfrequencyms
+
+Drivers should set the default heartbeatFrequencyMS to 5ms in order to take into account the latest changes regarding streaming protocol. If a test has an explicit heartbeatFrequencyMS value, drivers should use the explicit value.
```

Comment:
I'm on the fence about sprinkling this throughout all of the test files. It seems like something that would be better documented in a "best practices" document at the top of the repository, since it might as well apply to any test runner working with a server. Drivers would also probably want to do this in their own test suites (I've done so in my PHPUnit configuration for a few years now).

Reply:
>a "best practices" document
do you mean this doc: https://github.com/mongodb/specifications/blob/master/README.rst ?

Reply:
The main README may be the best place for now. I'll defer to @jyemin and @ShaneHarvey, though.

My main concern is wanting to avoid duplicating the same text in multiple places.

Reply:
I think this guidance would be out of place in the main readme. What if we add the description of "Speeding Up Tests" to only the transaction spec file and then link to that section in the other files?

Reply:
sgtm, @jmikola ? :)

Reply:
I don't object. I'd also suggest noting this in SPEC ticket and linking it up to [DRIVERS-709](https://jira.mongodb.org/browse/DRIVERS-709). From there, I can include it in the final README for the unified test runner.

Reply:
done